packages/next-app/src/app/internalmanager/(components)/TokenSelect.tsx

```diff
@@ -156,15 +172,100 @@ function TokenModal({
     }
   }
 
+  const tokenExplorerUrl = buildExplorerTokenURL({
+    chainId: chain!.id,
```

Comment:
#could make this not required and throw an error or return empty if `chain?.id` is not present
```suggestion
    chainId: chain.id,
```